benchmarks/continuous.py

```diff
@@ -89,15 +91,53 @@ def numpy_sum():
   return np.sum(xs * xs)
 
 
+def numpy_gaussian(n):
+  return lambda: np.random.normal(size=n)
+
+
+def numpy_matmul(n, width):
+  m1 = np.random.normal(size=(width, n, n)).astype(np.float32)
+  m2 = np.random.normal(size=(width, n, n)).astype(np.float32)
+  return lambda: np.matmul(m1, m2)
+
+
+def numpy_matvec(n, width):
+  ms = np.random.normal(size=(width, n, n)).astype(np.float32)
+  vs = np.random.normal(size=(width, n)).astype(np.float32)
+  return lambda: np.einsum('...ij,...j', ms, vs)
```

Comment:
Perhaps include unbatched examples too?

Reply:
Do you mean special-case to width=1 here?  Actually, is there a better function to test against than `np.einsum`?